tests/setup_test.py

```diff
@@ -1,8 +1,9 @@
+import logging
 import sys
 import os
 
 
 PACKAGE_PARENT = '..'
 SCRIPT_DIR = os.path.dirname(os.path.realpath(os.path.join(os.getcwd(), os.path.expanduser(__file__))))
-print(os.path.normpath(os.path.join(SCRIPT_DIR, PACKAGE_PARENT)))
+logging.log(logging.INFO,  os.path.normpath(os.path.join(SCRIPT_DIR, PACKAGE_PARENT)))
```

Comment:
There are two spaces after `logging.log(logging.INFO,` part in all changes in this PR. Maybe consider using one space instead?

Reply:
good catch, I'll fix. Thanks